rhel8/profiles/ospp.profile

```diff
@@ -9,839 +9,113 @@ description: |-
 
 
-    ### FCS_DTLS_EXT.1.2: https://www.niap-ccevs.org/MMO/PP/-424-/#FCS_DTLS_EXT.1.2
-    ### The OS shall implement the requirements in TLS (FCS_TLSC_EXT.1) for the DTLS
-    ### implementation, except where variations are allowed according to DTLS
-    ### 1.2 (RFC 6347).
 
-    #######################################################
-    ## FCS_TLSC_EXT.2 TLS Client Protocol
 
-    ### FCS_TLSC_EXT.2.1: https://www.niap-ccevs.org/MMO/PP/-424-/#FCS_TLSC_EXT.2.1
-    ### The OS shall present the Supported Elliptical Curves Extension in the Client
-    ### Hello with the following NIST curves: [selection: secp256r1, secp384r1,
-    ### secp521r1].
 
 
-    #######################################################
-    # Appendix C Objective Requirements
 
-    #######################################################
-    ## FCS_TLSC_EXT.3 TLS Client Protocol
 
-    ### FCS_TLSC_EXT.3.1: https://www.niap-ccevs.org/MMO/PP/-424-/#FCS_TLSC_EXT.3.1
-    ### The OS shall present the signature_algorithms extension in the Client Hello
-    ### with the supported_signature_algorithms value containing the following hash
-    ### algorithms: [selection: SHA256, SHA384, SHA512] and no other hash algorithms.
 
 
-    #######################################################
-    ## FPT_SRP_EXT.1 Software Restriction Policies
+## File system settings
 
-    ### FPT_SRP_EXT.1.1: https://www.niap-ccevs.org/MMO/PP/-424-/#FPT_SRP_EXT.1.1
-    ### The OS shall restrict execution to only programs which match an
-    ### administrator-specified [selection:
-    ###     - file path,
-    ###     - file digital signature,
-    ###     - version,
-    ###     - hash,
-    ###     - [assignment: other characteristics]]
 
 
-    #######################################################
-    ## FPT_W^X_EXT.1 Write XOR Execute Memory Pages
 
-    ### FPT_W^X_EXT.1.1: https://www.niap-ccevs.org/MMO/PP/-424-/#FPT_W^X_EXT.1.1
-    ### The OS shall prevent allocation of any memory region with both write
-    ### and execute permissions except for [assignment: list of exceptions]
 
 
 
-
-
-
-
-
-
-    #######################################################
-    ## Rules that need a home?
-
-    - enable_fips_mode
-    - sysctl_kernel_yama_ptrace_scope
-    - sysctl_kernel_kptr_restrict
-    - sysctl_kernel_kexec_load_disabled
-    - sysctl_kernel_dmesg_restrict
-    - grub2_slub_debug_argument
-    - grub2_page_poison_argument
-    - grub2_vsyscall_argument
-
-    - audit_rules_privileged_commands_at
-    - audit_rules_privileged_commands_crontab
-    - audit_rules_privileged_commands_mount
-    - audit_rules_privileged_commands_umount
-    - audit_rules_sysadmin_actions
-    - audit_rules_session_events
-
-    - audit_rules_privileged_commands_ssh_keysign
-    - rsyslog_cron_logging
-    - mount_option_dev_shm_nodev
-    - mount_option_dev_shm_noexec
-    - mount_option_dev_shm_nosuid
-    - package_abrt_removed
-
     - var_system_crypto_policy=fips
```

Comment:
`fips` selection for crypto policy won't do. The set of ciphers, MACs and other algorithms are different.